src/android/BackgroundGpsPlugin.java

```diff
@@ -84,6 +93,7 @@ public boolean execute(String action, JSONArray data, CallbackContext callbackCo
                 this.notificationTitle = data.getString(8);
                 this.notificationText = data.getString(9);
                 this.stopOnTerminate = data.getString(11);
+                this.postLocationsToServer = data.getBoolean(12);
```

Comment:
Could this #postLocationsToServer param not be inferred from the absence of #url param?